storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -800,6 +802,12 @@ struct my_option xb_client_options[] = {
      (G_PTR *)&xtrabackup_use_free_memory_pct,
      (G_PTR *)&xtrabackup_use_free_memory_pct, 0, GET_UINT, REQUIRED_ARG, 0, 0,
      100, 0, 1, 0},
+    {"calculate-memory", OPT_XTRA_CALCULATE_MEMORY,
+     "This option enable/disable the calculation of memory required to prepare "
+     "the backup. The calculation happens during backup. (Default OFF)",
+     (G_PTR *)&xtrabackup_calculate_memory,
+     (G_PTR *)&xtrabackup_calculate_memory, 0, GET_BOOL, NO_ARG, 0, 0, 0, 0, 0,
```

Comment:
@altmannmarcelo I remember some discussion around name. what is your opinion on estimate-memory?

Reply:
@satya-bodapati I have not strong opinion about it. estimate-memory sounds reasonable too. I will adjust

Reply:
@altmannmarcelo Am ok eitherway too. Please let me know (to proceed with review)

Reply:
Renamed